recipe/meta.yaml

```diff
@@ -13,8 +13,6 @@ source:
     - patches/fix.patch
 
 build:
-  # Skip for the `dev` branch creation
-  skip: true
   number: 0
```

Comment:
For reference, those builds target the `dev` branch which uses a specific label, `conda-forge/label/spdlog_dev`:

https://github.com/conda-forge/spdlog-feedstock/blob/672266b0ac02963fdbd974eaca4b363044ed1e51/recipe/conda_build_config.yaml#L1-L2